public/app/core/components/TransformersUI/GroupingToMatrixTransformerEditor.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useCallback } from 'react';
+import {
+  DataTransformerID,
+  SelectableValue,
+  standardTransformers,
+  TransformerRegistryItem,
+  TransformerUIProps,
+} from '@grafana/data';
+import { Select } from '@grafana/ui';
+import { GroupingToMatrixTransformerOptions } from '@grafana/data/src/transformations/transformers/groupingToMatrix';
+import { useAllFieldNamesFromDataFrames } from './utils';
+
+export const GroupingToMatrixTransformerEditor: React.FC<TransformerUIProps<GroupingToMatrixTransformerOptions>> = ({
+  input,
+  options,
+  onChange,
+}) => {
+  const fieldNames = useAllFieldNamesFromDataFrames(input).map((item: string) => ({ label: item, value: item }));
+
+  const onSelectColumn = useCallback(
+    (value: SelectableValue<string>) => {
+      onChange({
+        ...options,
+        columnField: value.value,
+      });
+    },
+    [onChange, options]
+  );
+
+  const onSelectRow = useCallback(
+    (value: SelectableValue<string>) => {
+      onChange({
+        ...options,
+        rowField: value.value,
+      });
+    },
+    [onChange, options]
+  );
+
+  const onSelectValue = useCallback(
+    (value: SelectableValue<string>) => {
+      onChange({
+        ...options,
+        valueField: value.value,
+      });
+    },
+    [onChange, options]
+  );
+
+  return (
+    <div className="gf-form-inline">
+      <div className="gf-form gf-form--grow">
+        <div className="gf-form-label width-8">Column</div>
+        <Select
+          menuShouldPortal
+          options={fieldNames}
+          value={options.columnField}
+          onChange={onSelectColumn}
+          isClearable
+        />
+      </div>
+      <div className="gf-form gf-form--grow">
+        <div className="gf-form-label width-8">Row</div>
+        <Select menuShouldPortal options={fieldNames} value={options.rowField} onChange={onSelectRow} isClearable />
+      </div>
+      <div className="gf-form gf-form--grow">
+        <div className="gf-form-label width-8">Cell Value</div>
+        <Select menuShouldPortal options={fieldNames} value={options.valueField} onChange={onSelectValue} isClearable />
+      </div>
+    </div>
```

Comment:
think specifying the `gf-form` classes is deprecated now, take a look at https://github.com/grafana/grafana/blob/main/public/app/features/transformers/prepareTimeSeries/PrepareTimeSeriesEditor.tsx#L67 for a nicer way 👍